test/test_torch.py

```diff
@@ -1708,12 +1709,49 @@ def test_clamp(self):
         torch.clamp(m1, max=max_val, out=out)
         self.assertEqual(out, res1)
 
+        # if the tensor contains nan case
+        test_tens = torch.FloatTensor([float('nan')])
```

Comment:
This line can be modified like this: `test_tens = torch.tensor([nan], device=device)`. `nan` will work here since it is explicitly imported.